Source/UnitTests/Core/DSP/DSPTestBinary.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+#include <array>
+
+static const std::vector<u16> s_dsp_test_bin = {
```

Comment:
Ah yeah, I moved on and forgot about putting those in their own sources files. Nice catch, fixed.

I would have preferred to stick them each in a `std::array,` but DSPDisassembler requires vector arguments, so they'd have to be turned into vectors anyways :/ It'd be nice to incorporate something like GSL's `span` type at some point.